react/tic-tac-toe/src/app/screens/Game/components/Square/index.js

```diff
@@ -1,11 +1,26 @@
 import React from 'react';
+import PropTypes from 'prop-types';
```

Comment:
Since this component is only used inside Row, then it should be placed in `Row/components/Square`